cassandra/src/main/java/org/hibernate/ogm/datastore/cassandra/impl/configuration/CassandraConfiguration.java

```diff
@@ -15,7 +15,7 @@
  *
  * @author Jonathan Halliday
  */
-public class CassandraConfiguration extends DocumentStoreConfiguration {
```

Comment:
Ok, I will send a separate pull requests for this, It should be already a set of separate commit